### PR TITLE
[ja] Add a feature section content "self-healing"

### PR DESCRIPTION
**Pull request by IbukiMnw**
Add a missing feature sections content in "ja". the title of new content is "self-healing" in "en".

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@IbukiMnw Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

**Comment by dipesh-rawat** [comment]
/retitle [ja] Add a feature section content "self-healing"

**Comment by atoato88** [comment]
The URL for the page on deploy preview for reviewing the content updated:
- https://deploy-preview-43979--kubernetes-io-main-staging.netlify.app/ja/#features

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 title: ReplicaSet
+feature:
+  title: セルフヒーリング
+  anchor: ReplicaSetがどのように動くか {#how-a-replicaset-works}
+  description: >
+    起動に失敗したコンテナを再起動し、ノードが停止したときにコンテナを置き換えて再スケジュールし、
+    ユーザーが定義したヘルスチェックに応答しないコンテナを停止させ、サービスの準備が出来るまではそれらのコンテナをクライアントに示しません。
 content_type: concept
-weight: 20
+weight: 30
```
Comment (atoato88) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:11]: This value is 20 on [the content of `en`](https://raw.githubusercontent.com/kubernetes/website/523b67d55ecfdcf5c86b0707bcd3fc1207fd7fb8/content/en/docs/concepts/workloads/controllers/replicaset.md).
What is the reason for updating this value with `30`?

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 ---
 reviewers:
 title: ReplicaSet
+feature:
+  title: セルフヒーリング
+  anchor: ReplicaSetがどのように動くか {#how-a-replicaset-works}
```
Comment (atoato88) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:6]:
```suggestion
  anchor: how-a-replicaset-works
```
Current code renders `#replicasetがどのように動くか-how-a-replicaset-works` as anchor, so specify [the anchor ID included in the `ja` content.](https://github.com/kubernetes/website/pull/43979/files#diff-82b9d0bd19cdaa0d7c2d162550e096aef1049b0542f064395a7348c517b4f320R21)

**Review by atoato88 — COMMENTED** [review]
Please check the comments.

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by atoato88** [comment]
@IbukiMnw 
Sorry for mention you, how about the sign to the CLA and updating the content?

**Comment by atoato88** [comment]
@IbukiMnw 
Gentle ping.

Is there any problem about signing the CLA?
If so, please refer [this article](https://github.com/kubernetes-sigs/contributor-playground/blob/master/japan/assets/attendee-prerequisites.md#cncf-cla-%E3%81%B8%E3%81%AE%E3%82%B5%E3%82%A4%E3%83%B3%E3%82%A2%E3%83%83%E3%83%97) for checking the CLA process in Japanese.

Besides, please feel free to contact me.

**Comment by Gauravpadam** [comment]
Closing this as the CLA status is still red
Please feel free to re-open this

/close

**Pull request closed without merging**